src/compiler/crystal/tools/doc/type.cr

```diff
@@ -166,7 +166,7 @@ class Crystal::Doc::Type
             defs << method(def_with_metadata.def, false)
           end
         end
-        stable_sort! defs, &.name.downcase
+        defs.sort_by!(stable: true, &.name.downcase)
```

Comment:
Since stable is default, the argument is technically not necessary. I don't mind keeping it explicit, though.

Reply:
Here's `stable` is meaningful and essential. When sorting implementation change other, it will be broken if `stable: true` is missing. Thus, I set `stable: true`.

@HertzDevil What's your opinion?